internal/app/singularity/remote_get_login_password_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2018-2023, Sylabs Inc. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2023, Sylabs Inc. All rights reserved.
```

This code is new, so only 2023 is relevant here.